src/particles/plasma/PlasmaParticleContainerInit.cpp

```diff
@@ -38,10 +38,28 @@ InitParticles (const amrex::IntVect& a_num_particles_per_cell,
     const amrex::Real scale_fac = Hipace::m_normalized_units?
                                   1./num_ppc : dx[0]*dx[1]*dx[2]/num_ppc;
 
+
+    amrex::IntVect box_nodal{amrex::IndexType::CELL,amrex::IndexType::CELL,amrex::IndexType::CELL};
+    amrex::IntVect box_grow{0, 0, 0};
+    amrex::Real x_offset = 0._rt;
+    amrex::Real y_offset = 0._rt;
+
+    if (ParticleGeom(lev).Domain().length(0) % 2 == 1 && a_num_particles_per_cell[0] % 2 == 1) {
+        box_nodal[0] = amrex::IndexType::NODE;
+        box_grow[0] = -1;
+        x_offset = -0.5_rt;
+    }
```

Comment:
Are the values stored in `box_grow` used to grow both the lower and the upper bound of the box? Or just 1 side?
Just to make sure we preserve the symmetry: with e.g. 64 cells, there are 65 nodes, and we want 63 particles on the nodes (except for each boundary), or (optional, definitely not same priority but maybe useful in the future) 64 with periodic boundary conditions (where there the particle is added to either the upper or the lower end, as they should be the same).

Reply:
It changes both lower and upper bound so it will be 63 * ppc particles for that example